docs/swagger/traPortfolio.v1.yaml

```
@@ -56,12 +62,197 @@ paths:
           description: Bad Request
         '404':
           description: Not Found
+      description: ユーザー情報を修正します
+      tags:
+        - user
       requestBody:
         content:
           application/json:
             schema:
               $ref: '#/components/schemas/EditUserRequest'
-      description: ユーザー情報を修正します
+  '/user/{userId}/projects':
+    parameters:
+      - $ref: '#/components/parameters/userIdInPath'
+    get:
+      summary: ユーザーが所属するプロジェクトの取得
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Project'
+        '404':
+          description: Not Found
+      operationId: getUsersProject
+      description: ユーザーが所属している（いた）プロジェクトを取得します
+      tags:
+        - user
+        - user project
+  /projects:
+    get:
+      summary: プロジェクトのリストを取得
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Project'
+      operationId: getProjects
+      description: プロジェクトのリストを取得します
+      tags:
+        - project
+    post:
+      summary: プロジェクトの作成
+      operationId: postProject
+      responses:
+        '200':
+          description: OK
+      description: プロジェクトを作成します
+      tags:
+        - project
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PostProjectRequest'
```

```suggestion
              $ref: '#/components/schemas/PostProject'
```
Postのレスポンスは基本何もないか`Project`みたいな何も修飾子なしのものだからリクエストのほうに`Request`をつけると冗長な気がする